nix/build.nix

```diff
@@ -38,7 +38,7 @@
   nodejs_22,
 
   withGLES ? false,
-  profile ? "release",
+  profile ? "dev",
```

Comment:
Just checking, is the change to the profile here intentional?

Reply:
nope, good catch!